### ISSUE TITLE
Timetable block - The first day of the week cannot be changed

### ISSUE THREAD

**Issue by acquaalta** [bdecentgmbh/moodle-block_timetable#1]
Hi,
It seems that the "Timetable" block ignores the "calendar_startwday" ("Start of week") setting, and also does not have an equivalent setting itself. Therefore, when using the "This week" view, we still see Monday as the first day of the week, instead of Sunday.
Is it something that can be solved?

I would like to add that this plugin has a lot of added value - It takes into consideration activities that the "Timeline" block ignores (Non-Core activities) and it also offer the horizontal view.

![TimetableBlock-FirstDayOfTheWeek](https://user-images.githubusercontent.com/51836675/158028132-6ab50d32-a6b6-4742-9bc2-9924418711b4.jpg)

**Comment by acquaalta** [comment]
OK, I think I found the issue. 

TL;DR: Make sure that the "First day of week" in calendar preferences of the user profile is set correctly, and even if so, try to re-save it.

Although I changed the "calendar_startwday" setting from Monday to Sunday, and although that when I entered the calendar preferences of my profile/user (User menu -> Profile -> Calendar preferences) I saw that "Sunday" was set to be the first day of the week, it didn't come into effect until I changed the setting in my profile to another day, saved, changed back to Sunday and then saved again.
Maybe it was just a "refresh" issue, and something that should not be relevant in Moodle environments that their general first day of the week setting would be set correctly before the "Timetable" block would be installed and before users will be created (besides the admin).

Since this is not actually an issue that is related to the plugin itself, I guess that you can close it or delete it :-)
Sorry for the interruption.
